load-gen/src/main/java/io/smartcat/data/loader/api/Terminateable.java

```diff
@@ -0,0 +1,13 @@
+package io.smartcat.data.loader.api;
+
+/**
+ * Has ability to terminate itself.
+ */
+public interface Terminateable {
```

Comment:
No, I think it is good to switch, but also to remove closing of those resources from load generator. Since it seems to me that it is not reponsibility of LoadGenerator to do that, because those objects are created outside of it.